Formula/libpsl.rb

```diff
@@ -0,0 +1,29 @@
+class Libpsl < Formula
+  desc "C library for the Publix Suffix List"
+  homepage "https://github.com/rockdaboot/libpsl"
```

Comment:
18 stars, 5 watchers, no forks. Please explain why this package is notable, or we can't accept it in core. Ref: https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Acceptable-Formulae.md.